cmrtlib/linux/CMakeLists.txt

```diff
@@ -79,7 +79,8 @@ set(CMRT_DEFINES "")
 cmrt_include_directory(${CMAKE_CURRENT_LIST_DIR}/../agnostic)
 cmrt_include_directory(${CMAKE_CURRENT_LIST_DIR}/../linux)
 
-setup_library(igfxcmrt "${CMRT_SOURCES}" TRUE "igfxcmrt${TARGET_MODIFIER}")
+#setup_library(igfxcmrt "${CMRT_SOURCES}" TRUE "igfxcmrt${TARGET_MODIFIER}")
```

Comment:
don't comment - delete if you don't need this.

Reply:
yes, delete it.

Reply:
I did fix it in my last update. Can you review it in the latest file changes?